sdks/java/io/amazon-web-services/src/main/java/org/apache/beam/sdk/io/aws/dynamodb/AwsClientsProvider.java

```diff
@@ -28,7 +28,11 @@
  * ensure it can be sent to worker machines.
  */
 public interface AwsClientsProvider extends Serializable {
-  AmazonCloudWatch getCloudWatchClient();
+  @Deprecated
```

Comment:
Could you add a comment for users why it's deprecated?

Reply:
👍